source/Generic/GamesSizeCalculator/GamesSizeCalculator.cs

```diff
@@ -279,16 +295,38 @@ public override void OnLibraryUpdated(OnLibraryUpdatedEventArgs args)
             SavePluginSettings(settings.Settings);
         }
 
+        private ICollection<IOnlineSizeCalculator> GetOnlineSizeCalculators(SteamApiClient steamClient)
+        {
+            var onlineSizeCalculators = new List<IOnlineSizeCalculator>();
+            if (settings.Settings.GetUninstalledGameSizeFromSteam)
+            {
+                onlineSizeCalculators.Add(new SteamSizeCalculator(steamClient, GetDefaultSteamAppUtility()
+                    , settings.Settings.IncludeDlcInSteamCalculation
+                    , settings.Settings.IncludeOptionalInSteamCalculation
+                    , settings.Settings.GetSizeFromSteamNonSteamGames
+                    , settings.Settings.DepotRegionWords.ToArray()
+                    , settings.Settings.DepotRegionWordsBlacklist.ToArray()));
+            }
```

Comment:
Is it possible that instead of passing all those settings values one by one, pass the whole settings object instead and handle it in the constructor to keep it cleaner perhaps?